scripts/services/check-removed-services.js

```diff
@@ -0,0 +1,133 @@
+const fs = require('fs/promises');
+const path = require('path');
+const yaml = require('js-yaml');
+
+const YML_FILE_EXTENSION = '.yml';
+
+const servicesDir = path.resolve(__dirname, '../../services/');
+
+/**
+ * Converts a service name to lowercase and replaces special characters.
+ *
+ * @param {string} serviceName - The service name to be normalized.
+ * @returns {string} The normalized service name.
+ */
+const normalizeFileName = (serviceName) => serviceName.toLowerCase().replace(/[^a-z0-9.]/g, '');
+
+/**
+ * Reads and parses the services data from from a JSON file.
+ *
+ * @param {string} jsonFilePath - The path to the JSON file.
+ * @returns {Promise<object[]|null>} A promise that resolves to an array of services data objects from JSON file.
+ * if the JSON file is successfully read and parsed. Returns `null` if there's an error during the process.
+ */
+const getJsonObjects = async (jsonFilePath) => {
+    try {
+        const jsonFileContent = await fs.readFile(jsonFilePath);
+        const jsonObjects = JSON.parse(jsonFileContent);
+        return jsonObjects.blocked_services;
+    } catch (error) {
+        console.error('Error while reading JSON file:', error);
```

Comment:
Maybe it would be good to log file path here

Reply:
we have only one `json` - `services.json`, I can add it to console.log. before the `json` check is performed inside `checkRemovedServices` we `validateJson`, leave two checks?

Reply:
Ok then